components/scream/src/share/field/field_request.hpp

```diff
@@ -2,7 +2,7 @@
 #define SCREAM_FIELD_REQUEST_HPP
 
 #include "share/field//field_identifier.hpp"
-#include "share/scream_types.hpp"
+#include "share/util//scream_utils.hpp"
```

Comment:
Maybe this is a feature I don't know, but should there be 2 slashes here?

Reply:
I've seen that same typo happening multiple times on my PRs. It might be the autocomplete and me having a race condition for adding the slash, I am not sure... But no, it's not a feature. :)